src/lib/tls/tls_magic.h

```diff
@@ -50,18 +50,15 @@ enum class Connection_Side {
    SERVER BOTAN_DEPRECATED("Use Connection_Side::Server") = Server,
 };
 
-// This will become an enum class in a future major release
-enum Record_Type {
-   INVALID            = 0,  // RFC 8446 (TLS 1.3)
-
-   CHANGE_CIPHER_SPEC = 20,
-   ALERT              = 21,
-   HANDSHAKE          = 22,
-   APPLICATION_DATA   = 23,
+enum class Record_Type: uint8_t {
+   Invalid            = 0,  // RFC 8446 (TLS 1.3)
 
-   HEARTBEAT          = 24, // RFC 6520 (TLS 1.3)
+   CCS                = 20,
```

Comment:
Might be a nit, but I'd prefer to expand the abbreviation as `ChangeCipherSpec` for readability.

Reply:
Went back and forth on this one myself. Will change to `ChangeCipherSpec`.

Reply:
Renamed